src/test/java/no/nav/pensjon/selvbetjeningopptjening/opptjening/EndringPensjonsbeholdningCalculatorTest.java

```diff
@@ -145,6 +145,38 @@ void when_beholdning_list_has_one_element_with_fomDateOnRegulationDate_with_lonn
         assertEquals(beholdning.getBelop(), endringer.get(2).getBeholdningsbelop());
     }
 
+    @Test
```

Comment:
Burde vi teste på sluttbeløpene og?

Reply:
Ser at en annen test her burde være dekkende for beløpene, så lar disse testene fokusere kun på det caset som trigger feilen.